app/subscriber/src/features/my-folders/ConfigureFolder.tsx

```diff
@@ -278,6 +279,34 @@ export const ConfigureFolder: React.FC<IConfigureFolderProps> = () => {
             });
           }}
         />
+        <Row className="keep-stories">
```

Comment:
This is a property of the folder, not the schedule.  It should have been implemented like the Editor Folder Admin.